action/action-util/src/main/java/com/powsybl/action/util/IdentifierScalable.java

```diff
@@ -0,0 +1,72 @@
+/**
+ * Copyright (c) 2017, RTE (http://www.rte-france.com)
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+package com.powsybl.action.util;
+
+import com.powsybl.commons.PowsyblException;
+import com.powsybl.iidm.network.Generator;
+import com.powsybl.iidm.network.Identifiable;
+import com.powsybl.iidm.network.Network;
+
+import java.util.List;
+import java.util.Objects;
+
+class IdentifierScalable extends AbstractScalable {
+
+
+    private final String id;
+
+    private Scalable scalable;
```

Comment:
You should not cache scalable because it depends of the network instance. A same id can be used for different connectable in two different networks.